genestack_client/data_importer.py

```diff
@@ -485,17 +485,17 @@ def create_reference_genome(self,
                                 parent=None,
                                 name=None,
                                 description='',
-                                sequence_urls=None,
+                                sequence_urls=[],
```

Comment:
[Never use a mutable default argiment](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments).
Also, why this change at all?

Reply:
Fixed. The idea here is that sequence files are no longer used in reference genome related functionality. I updated the docs, removed redundant metadata addition and left the parameter untouched for backwards compatibility. Also added an example script that can be used to add genomes